types/bootstrap/index.d.ts

```diff
@@ -223,7 +223,7 @@ export interface ScrollspyOption {
      *
      * @default ""
      */
-    target?: string | Element;
+    target?: string | JQuery | Element;
```

Comment:
`JQuery<Element>` may be better.

```suggestion
    target?: string | JQuery<Element> | Element;
```